scripts/acq_results.py

```diff
@@ -67,10 +69,10 @@ def _receive_acq_result(self, data):
     self.acqs.append({})
     a = self.acqs[-1]
 
-    a['SNR'] = struct.unpack('f', data[0:4])[0]  # SNR of best point.
-    a['CP'] = struct.unpack('f', data[4:8])[0]   # Code phase of best point.
-    a['CF'] = struct.unpack('f', data[8:12])[0]  # Carr freq of best point.
-    a['PRN'] = struct.unpack('B', data[12])[0]   # PRN of acq.
+    a['SNR'] = struct.unpack('f', data.payload[0:4])[0]  # SNR of best point.
```

Comment:
Yeah, wanted to leave this in initially before porting over to the new parsers. Believe there might be some other areas that could also benefit as well, but wanted a less risky iteration here.